forc/src/cli/commands/init.rs

```diff
@@ -5,20 +5,27 @@ use clap::Parser;
 /// Create a new Forc project.
 #[derive(Debug, Parser)]
 pub struct Command {
-    /// The default program type, excluding all flags or adding this flag creates a basic contract program.
+    /// The path at which to create the manifest
     #[clap(long)]
-    pub contract: bool,
-    /// Adding this flag creates an empty script program.
+    pub path: Option<String>,
+    /// Create a package with a binary target (src/main.sw). This is the default behavior
```

Comment:
```suggestion
    /// Create a package with a script target (src/main.sw).
```

I believe our default program type should be the `contract` program type. It looks like this is correct under the `forc new` CLI command, but not here.